index.html

```diff
@@ -32,92 +32,121 @@
     </div>
   </nav>
 
+  <!--Background image-->
   <div class="container-fluid pt-5 pb-5" id="bgImage">
-    <div class="container p-5 mx-auto bg-light">
-      <div class="row mx-auto">
-        <label for="location" class="form-label">Location:</label>
-        <div class="input-group mb-3">
-          <span class="input-group-text" id="basic-addon3"></span>
-          <input type="text" class="form-control" id="location" aria-describedby="basic-addon3">
-        </div>
 
-      </div>
-      <div class="row mx-auto">
-        <label for="dateTime" class="form-label">Date/Time:</label>
-        <div class="input-group mb-3">
-          <span class="input-group-text" id="basic-addon3"></span>
-          <input type="text" class="form-control" id="dateTime" aria-describedby="basic-addon3">
+      <!--Inputboxes container-->
+      <div class="container p-5 mx-auto bg-light" id="homepagebox">
+        <div class="row mx-auto">
+          <label for="location" class="form-label">Location:</label>
+          <div class="input-group mb-3">
+            <span class="input-group-text" id="basic-addon3"></span>
+            <input type="text" class="form-control" id="location" aria-describedby="basic-addon3">
+          </div>
+
         </div>
-      </div>
-      <div class="row mx-auto">
-        <div class="col-4">
-          <button type="button" class="btn btn-primary">Submit</button>
+        <div class="row mx-auto">
+          <label for="dateTime" class="form-label">Date/Time:</label>
+          <div class="input-group mb-3">
+            <span class="input-group-text" id="basic-addon3"></span>
+            <input type="text" class="form-control" id="dateTime" aria-describedby="basic-addon3">
+          </div>
+        </div>
+        <div class="row mx-auto">
+          <div class="col-4">
+            <button type="button" class="btn btn-primary">Submit</button>
+          </div>
         </div>
-      </div>
 
-    </div>
+      </div>
+ 
   </div>
+
+  <!--------------->
+  <!--Is there a more elegant way to solve this spacing issue?-->
```

Comment:
perhaps we can add margins to the bottom of the fluid container and then the top of the one below, I'll look more comprehensively later but this might work, at least looking at the code without full context.